arangod/Pregel/Conductor.cpp

```diff
@@ -748,28 +744,7 @@ bool Conductor::canBeGarbageCollected() const {
 
 void Conductor::collectAQLResults(VPackBuilder& outBuilder, bool withId) {
   MUTEX_LOCKER(guard, _callbackMutex);
-
-  if (_state != ExecutionState::DONE && _state != ExecutionState::FATAL_ERROR) {
```

Comment:
I'm really not sure why FatalError is here. I don't see any reason why results should be returned if the conductor is in a FatalError state. In my opinion this should not be the case, what do you think?
For now I just reproduced the behavior here and returned the results in the FatalError state but I can delete that easily.